File: Auto-generated-tasks/Difficulty-1/Task-4/Solution.cs
int NN = 6, d, Rez = 0;
int[] T = new int[NN];
for (d = 0; d < NN; d++) {
	if (T[d] > 0)
		Rez = Rez + 1;
}
Console.WriteLine("Rez=" + Rez);

File: Auto-generated-tasks/Difficulty-1/Task-4/Solution.go
package "main"
import "fmt"
NN:= 6
d:= 0
Rez:= 0
var T [NN]rune
for d = 0; d < NN; d++ {
	if (T[d] > 0)
		Rez = Rez + 1
}
fmt.PrintLn("Rez=", Rez)